pkg/users/dialog-utils.js

```diff
@@ -49,7 +49,7 @@ export function has_errors(errors) {
 function show_error_dialog(title, message) {
     const props = {
         id: "error-popup",
-        title: title,
+        title,
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18417-20230227-210817-602a6f78-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)